### PR TITLE
Signups #28

### PR DESCRIPTION
**Pull request by JamesHageman**
🙌 

connected to #28

### CODE REVIEW

**Comment by JamesHageman** [comment]
Need to adjust for https://github.com/georgeutsin/whistlet/pull/27

File: src/components/login/SignupForm.js [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+  if (!values.username) {
+    errors.username = 'You must provide a unique Username';
+  }
+
+  if (!values.password) {
+    errors.password = 'A password is required';
+  } else if (values.password.length < 6) {
+    errors.password = 'password should NOT be shorter than 6 characters';
+  } else if (values.confirmPassword !== values.password) {
+    errors.confirmPassword = 'passwords do not match!';
```
Comment (ethan-hardy) [inline src/components/login/SignupForm.js]: Nitpicky, but these should really be capitalized (ie "Passwords do not match!")

File: src/store.js [hunk 2]
```diff
@@ -15,9 +15,15 @@ export function configureStore(): Store {
 
   const sagaMiddleware = createSagaMiddleware();
 
+  let middleware = [ sagaMiddleware, logger ];
+
+  if (process.env.NODE_ENV === 'test') {
+    middleware = [ sagaMiddleware ];
+  }
```
Comment (ethan-hardy) [inline src/store.js]: Could we use a ternary operator here? 

`const middleware = process.env.NODE_ENV === 'test' ? [ sagaMiddleware ] : [ sagaMiddleware, logger ];`

This way we can make `middleware` `const`

**Comment by JamesHageman** [comment]
@ethan-hardy pls review again!

I still need to add polling for the taken usernames and emails, but the rest should be good to go

**Comment by ethan-hardy** [comment]
🐙 🐙 Looks good

**Comment by JamesHageman** [comment]
Still gotta do the email/username thing, so I won't mark 28 as done. I'll merge this anyway so you lads can create accounts.

**Pull request merged**